api/adaptive_load/input_variable_setter_impl.proto

```diff
@@ -0,0 +1,18 @@
+// Plugin-specific config protos for InputVariableSetter plugins.
+
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+// Note: RequestsPerSecondInputVariableSetter aka "rps" does not define a
+// custom config proto.
+
+// Configuration for an InputVariableSetter plugin that sets an HTTP header
+// within CommandLineOptions to a numeric value being varied by a
+// StepController.
+message NumericHttpHeaderInputVariableSetterConfig {
```

Comment:
I would suggest we focus on implementing the RPS plugin only in this set of PRs. We can add other plugin types and features in other set of PRs which will help us to stay focused and keep the reviews and test surface smaller.

Reply:
Agreed, removing this.